tests/search.test.ts

```diff
@@ -257,6 +257,51 @@ describe.each([
     expect(hit.id).toEqual(1)
   })
 
+  test(`${permission} key: search with _showRankingScore enabled`, async () => {
+    const client = await getClient(permission)
+
+    const response = await client.index(index.uid).search('prince', {
+      showRankingScore: true,
+    })
+
+    const hit = response.hits[0]
+
+    expect(response).toHaveProperty('hits', expect.any(Array))
+    expect(response).toHaveProperty('query', 'prince')
+    expect(hit).toHaveProperty('_rankingScore')
+  })
+
+  test(`${permission} key: search with showRankingScoreDetails enabled`, async () => {
+    const client = await getClient(permission)
+    const key = await getKey(permission)
+
+    await fetch(`${HOST}/experimental-features`, {
+      body: JSON.stringify({ scoreDetails: true }),
+      headers: {
+        Authorization: `Bearer ${key}`,
+        'Content-Type': 'application/json',
+      },
+      method: 'PATCH',
+    })
+
+    const response = await client.index(index.uid).search('prince', {
+      showRankingScoreDetails: true,
+    })
+
+    const hit = response.hits[0]
+
+    expect(response).toHaveProperty('hits', expect.any(Array))
```

Comment:
This assertion is not valid since you'll have a broken code anyway. In the line before you ensure you're taking the first item from an array :)

Reply:
I'm not asserting `hit` but `response`. Not sure why it's broken

Reply:
hits will always be an array, that's my point. Otherwise you'll have an exception!

Reply:
In your assertion, you're telling that you expect the hits key to be an array. And that will never fail.

Reply:
Oh yes, it's a test that tests Meilisearch not my SDK haha. It's a copy paste from other tests I can remove it